setup/module/api/policies/elb_group_policy.json

```diff
@@ -4,7 +4,29 @@
 	{
             "Effect": "Allow",
             "Action": [
-                "elasticloadbalancing:Describe*"
+                "elasticloadbalancing:ModifyListener",
```

Comment:
We should probably just do `"elasticloadbalancing:*"` if it's getting this large.

Reply:
Agreed ✅